spec/support/shared_examples/request_get_flexible_form.rb

```diff
@@ -7,71 +7,64 @@
 # If the user loads one of these forms, we change the workflow_state to match their request,
 # so the browser and the database agree about what form the user is currently using.
 RSpec.shared_examples "GET flexible form" do |form|
-  context "when a valid user is signed in" do
-    let(:user) { create(:user) }
 
-    before do
-      sign_in(user)
+  context "when a renewal is in progress" do
+    let(:transient_registration) do
```

Comment:
I see you have removed `sign_in(user)` from many tests. Is it because this is purely front-office functionality where users no longer need to sign-in?

Reply:
Yes. Ultimately we should be able to remove the devise and cancancan gems from the engine. It's not quite there yet.